ios/Source/Utils.swift

```diff
@@ -0,0 +1,6 @@
+
+public func createResolvers(with resolversId: NSString) -> (DidResolverProxy, SecretsResolverProxy) {
+    let didResolver = DidResolverProxy(resolversProxyModule: DIDCommResolversProxyModule(), resolversId: resolversId.asString)
```

Comment:
Do we create new instances of `DIDCommResolversProxyModule` for every resolver proxy?

Reply:
Decided to not store resolvers proxy module in `DidResolverProxy` and `SecretsResolverProxy` and, therefore, remove this instance creation.